qingstor/sdk/client/file_chunk.py

```diff
@@ -0,0 +1,27 @@
+import os
+from io import BytesIO
```

Comment:
import StringIO from compat instead of import directly.
Python2 do not have BytesIO